draft-ietf-jsonpath-base.md

```diff
@@ -1156,11 +1156,6 @@ relation-expr = comp-expr /                           ; comparison test
                 regex-expr                            ; regular expression test
 ~~~~
 
-Comparisons are restricted to Singular Path values and primitive values (such as number, string, `true`, `false`, `null`).
```

Comment:
Is the "singular path" requirement mentioned elsewhere?  I see it removed here and not added back anywhere.

Reply:
The semantics section actually implies that if a non-singular path was permissible, the result of the comparison would be "false", but the above sentence was actually talking about the syntax, so I've put it back. Thanks for spotting this.